src/component/common/InstanceStatus/InstanceStatus.tsx

```diff
@@ -14,9 +39,40 @@ export const InstanceStatus: FC = ({ children }) => {
                     <InstanceStatusBarMemo instanceStatus={instanceStatus!} />
                 )}
             />
+            <Dialogue
+                open={dialogOpen}
+                primaryButtonText="Upgrade trial"
+                secondaryButtonText={
+                    instanceStatus?.trialExtended
+                        ? 'Remind me later'
+                        : 'Extend trial (5 days)'
+                }
+                onClick={() => {
```

Comment:
This should eventually hit an API endpoint that extends the trial by 5 days. There should be a `/api/instance/extend` endpoint in auth-app.